config/locales/en.yml

```diff
@@ -155,6 +155,7 @@ en:
       intermediate_rent_product_name:
         blank: "Enter name of other intermediate rent product"
       saledate:
+        later_than_14_days_after: "Sale completion date must not be later than 14 days from today’s date"
```

Comment:
copy updates other than this one are not specific to this ticket, but keep the usage of "sale completion date" explicit and consistent.